tests/fixture/e2e/helpers.go

```diff
@@ -188,16 +186,34 @@ func WithSuggestedGasPrice(ethClient ethclient.Client) common.Option {
 	return common.WithBaseFee(baseFee)
 }
 
-// Verify that a new node can bootstrap into the network.
+// Verify that a new node can bootstrap into the network. This function is safe to call
+// from `Teardown` by virtue of not depending on ginkgo.DeferCleanup.
 func CheckBootstrapIsPossible(network *tmpnet.Network) {
+	require := require.New(ginkgo.GinkgoT())
+
 	if len(os.Getenv(SkipBootstrapChecksEnvName)) > 0 {
 		tests.Outf("{{yellow}}Skipping bootstrap check due to the %s env var being set", SkipBootstrapChecksEnvName)
 		return
 	}
 	ginkgo.By("checking if bootstrap is possible with the current network state")
 
-	node := AddEphemeralNode(network, tmpnet.FlagsMap{})
-	WaitForHealthy(node)
+	ctx, cancel := context.WithTimeout(context.Background(), DefaultTimeout)
+	defer cancel()
+
+	// Node stop will be initiated if the node is not started without error.
+	node, err := network.AddEphemeralNode(ctx, ginkgo.GinkgoWriter, tmpnet.FlagsMap{})
+	require.NoError(err)
+
+	err = tmpnet.WaitForHealthy(ctx, node)
```

Comment:
nit: could we add a comment that explains this error is checked later?
alternatively could put the stop logic below in a defer func but I'm not sure that is more readable.

Q: Was this node previously never stopped?